src/actions/topics/votes/create.cr

```diff
@@ -1,11 +1,6 @@
 class Topics::Votes::Create < BaseAction
   nested_action do
-    VoteForm.save params do |form, vote|
-      if vote
-        redirect to: Topics::Index
-      else
-        redirect to: Topics::Index
-      end
-    end
+    VoteForm.new(topic_id: topic_id).save
```

Comment:
You can instantiate a form with a NamedTuple and it will create a params object for you automatically. In the future this will also be allowed with `save!` 

It also uses `topic_id` which is automatically generated because this route is guaranteed to have that param